### PR TITLE
CB-8927 Add media to cspMeta and Windows as plat

### PR DESCRIPTION
**Pull request by muratsu**
Fixes CSP errors on media test.
Addes CSPs for Windows platform

### CODE REVIEW

File: www/csp-incl.js [hunk 1]
```diff
@@ -49,7 +49,13 @@ if (!window._disableCSP) {
...
         case 'android':
         case 'ios':
-            cspMetaContent = 'default-src \'self\' https://ssl.gstatic.com/accessibility/javascript/android/; connect-src \'self\' http://cordova-filetransfer.jitsu.com;frame-src \'self\' data: gap:; img-src \'self\' data:; style-src \'self\' \'unsafe-inline\'';
+        case 'windows8':
+            cspMetaContent = 'default-src \'self\' https://ssl.gstatic.com/accessibility/javascript/android/;' +
+                            ' connect-src \'self\' http://cordova-filetransfer.jitsu.com;' +
+                            ' media-src \'self\' http://cordova.apache.org/downloads/;' +
+                            ' frame-src \'self\' data: gap:;' +
+                            ' img-src \'self\' data:;' +
+                            ' style-src \'self\' \'unsafe-inline\'';
```
Comment (dblotsky) [inline www/csp-incl.js]: Even though the fact that this fixes things on Windows is actually really valuable, the formatting is the best thing about this change. Small nit: might be better to use double-quotes for the strings so you won't have to escape as much.

**Comment by nikhilkh** [comment]
LGTM.

**Comment by dblotsky** [comment]
Merged. Sorry, forgot to say "This closes #127." You may safely close and delete.

**Comment by nikhilkh** [comment]
You need the nifty coho merge-pr script. :wink:

**Comment by nikhilkh** [comment]
@muratsu Can you please close this PR?

**Pull request closed without merging**